src/Frontend/Page/Overview.hs

```diff
@@ -175,6 +175,7 @@ instance ToHtml Tabs where
 -- * idea lists
 
 data ListItemIdeaContext  -- TODO: rename to 'ListLocation'?  'Context' is used for too many things.
+                          -- 'Position'?  i think my favorite so far would be `WhatList`.  ~~mf
```

Comment:
Other solutions:
 * Use 3 different functions for the very similar task
 * Use 3 different types for the very similar task

And abstract out the common HTML generation parts.